emission/analysis/modelling/tour_model/similarity.py

```diff
@@ -41,10 +41,10 @@ def __init__(self, data, radius, old=True):
                 # print "a is %s" % a
                 t = a
                 try:
-                    start_lon = t.start_loc["coordinates"][0]
-                    start_lat = t.start_loc["coordinates"][1]
-                    end_lon = t.end_loc["coordinates"][0]
-                    end_lat = t.end_loc["coordinates"][1]
+                    start_lon = t.start_place.location["coordinates"][0]

```

Comment:
I did not sorry... I'm writing tests now...

On Sun, Apr 24, 2016 at 10:34 PM, shankari <notifications@github.com> wrote:

> In emission/analysis/modelling/tour_model/similarity.py
> <https://github.com/e-mission/e-mission-server/pull/263#discussion_r60864063>
> :
>
> > @@ -41,10 +41,10 @@ def __init__(self, data, radius, old=True):
> >                  # print "a is %s" % a
> >                  t = a
> >                  try:
> > -                    start_lon = t.start_loc["coordinates"][0]
> > -                    start_lat = t.start_loc["coordinates"][1]
> > -                    end_lon = t.end_loc["coordinates"][0]
> > -                    end_lat = t.end_loc["coordinates"][1]
> > +                    start_lon = t.start_place.location["coordinates"][0]
>
> I added some logging statements, and it is pretty clear that the one in
> representatives is the one being called. Which makes sense because
> t.start_place is an object ID, so I would expect t.start_place.location to
> fail.
>
> How did you test this to confirm that it works?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/263/files/599f341f0bacc3ce6e5843b5c861a0ff7710a049..7ece7b1184adb62db2c6d495bc91e633423750a7#r60864063>
>